src/server/game/Battlegrounds/ArenaTeam.cpp

```diff
@@ -967,3 +964,82 @@ ArenaTeamMember* ArenaTeam::GetMember(uint64 guid)
 
     return nullptr;
 }
+
+uint8 ArenaTeam::GetReqPlayersForType(uint32 type)
+{
+    if (!ArenaReqPlayersForType.count(type))
+    {
+        sLog->outError("FATAL: Unknown arena type %u!", type);
+        return 0xFF;
+    }
+
+    return ArenaReqPlayersForType[type];
+}
+
+void ArenaTeam::CreateTempArenaTeam(std::vector<Player*> playerList, uint8 type, std::string const& teamName)
+{
+    uint32 PlayerCountInTeam = static_cast<uint32>(playerList.size());
```

Comment:
can use `auto` in cases like this, no reason to repeat `uint32` twice in the same line